### PR TITLE
[s][AprilFools] Spacemen now make sounds when talking.

### PR DESCRIPTION
**Pull request by lordpidey**
Adds a quick and dirty speech for spacemen.  It only applies to carbon/humans. (So lizards/skeletons/flymen will all have it).

Honestly, this is a 15 min hack job, the code isn't polished, but it works well enough for a joke testmerge on april first.

To hear what it sounds like in practice, https://www.dropbox.com/s/ragns8izhwxgl80/spaceman%20sound.ogg?dl=0

I made all of the sounds personally, so no copyright memes apply.

### CODE REVIEW

**Comment by xxalpha** [comment]
This shouldn't be a joke feature, this should be a permanent feature.

**Comment by lordpidey** [comment]
If people want it to stay after april 1st, I'm fine with that.  But the code will need to be cleaned up before then.

**Comment by Cheridan** [comment]
my implementation is superiar

**Comment by Cyberboss** [comment]
Why don't you reopen it for april fools @Cheridan

**Comment by lordpidey** [comment]
Honestly Cheridan, mine produces much more dynamic sounds rather than the same repeating memememememe.  Now, I'll grant that your code is better.

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -142,6 +142,15 @@ var/list/crit_allowed_modes = list(MODE_WHISPER,MODE_CHANGELING,MODE_ALIEN)
 	log_say("[name]/[key] : [message]")
 	return 1
 
+/mob/living/carbon/human/say(message, bubble_type,var/list/spans = list(), sanitize = TRUE)
+	if(..())
+		spawn(0)
+			var/loops = min(length(message)/5,20)
+			for(var/i = 0 ; i<loops;i++)
```
Comment (ChangelingRain) [inline code/modules/mob/living/say.dm]: for(var/i in 1 to loops

**Comment by lzimann** [comment]
it was fun... once...
please stop

**Comment by lzimann** [comment]
Plus we are closing all april fools PRs, since they dont need to be open for a testmerge. Just don't delete the branch.

**Comment by TrustyGun** [comment]
https://www.youtube.com/watch?v=2uhxrPhVO8Y

File: code/modules/mob/living/say.dm (hunk 1 above)
Comment (lordpidey) [inline code/modules/mob/living/say.dm]: To clarify, this has been fixed in the branch.  It just doesn't show as fixed because the PR is closed.

**Pull request closed without merging**